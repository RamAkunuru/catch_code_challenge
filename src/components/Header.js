import React from 'react';
import Logo from './Logo';
import PriceDropdown from './PriceDropdown';

export default () => (
    <div className="header">
        <h1><Logo width="13.2rem" /></h1>
        <PriceDropdown />
    </div>
)
