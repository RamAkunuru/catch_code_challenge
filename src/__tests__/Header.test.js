import React from 'react';
import { shallow } from 'enzyme';
import Header from '../components/Header';

describe("Header component", () => {
    test("render matches the snapshot", () => {
        const component = shallow(<Header />);
        expect(component).toMatchSnapshot();
    });
});
