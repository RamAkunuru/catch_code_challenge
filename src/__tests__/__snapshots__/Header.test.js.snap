// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component render matches the snapshot 1`] = `
<div
  className="header"
>
  <h1>
    <Logo
      width="13.2rem"
    />
  </h1>
  <Connect(PriceDropdown) />
</div>
`;
